### DISCUSSION TITLE
HealthMultiplier not working in NPC prefabs

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#15068] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

It appears that the healthmultiplier attribute in NPC prefab is not working. 
Example: an outpost HR manager that should have a health multiplier of 3:
![image](https://github.com/user-attachments/assets/47c0087d-e241-4bf0-9ef3-ef5c3a51d221)
![image](https://github.com/user-attachments/assets/8ee6a1db-6b36-4eb9-a7c6-3cb683e77ecc)

### Reproduction steps

- hurt outpost NPCs with a healthmultiplier of 3
- they turn out to have the same amount of vitality as a normal crew member

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Could be related to [this ](https://new.reddit.com/r/Barotrauma/comments/1ghh41d/final_boss_bug/)
I assume that person never reported that on github
And i'm too lazy to test it myself

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/609ddc58d78a8a70e635ba3c16ebb1ef744f5129 (commit in our private repo). Fix coming up in a future update.
